src/lib.rs

```diff
@@ -5,12 +5,13 @@
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
 #![no_std]
-#![cfg_attr(feature = "alloc", warn(missing_docs), doc = include_str!("../README.md"))]
+#![doc = include_str!("../README.md")]
+#![deny(missing_docs)]
 
-#[cfg(feature = "alloc")]
 extern crate alloc;
```

Comment:
Do we use the `alloc` crate explicitly at all?

Reply:
Actually we don't, good catch! I removed it